src/composables/useFlaw.ts

```diff
@@ -1,17 +1,29 @@
-import { ref } from 'vue';
+import { ref, readonly, computed } from 'vue';
 
 import type { ZodFlawType } from '@/types/zodFlaw';
+import { deepCopyFromRaw } from '@/utils/helpers';
 
 const flaw = ref<ZodFlawType>(blankFlaw());
+const initialFlaw = ref<ZodFlawType>(blankFlaw());
 const relatedFlaws = ref<ZodFlawType[]>([]);
 
 function resetFlaw() {
   flaw.value = blankFlaw();
+  initialFlaw.value = blankFlaw();
   relatedFlaws.value = [];
 }
 
+function setFlaw(fetchedFlaw: ZodFlawType) {
+  initialFlaw.value = deepCopyFromRaw(fetchedFlaw);
+  flaw.value = fetchedFlaw;
+}
+
+const isFlawUpdated = computed(
+  () => JSON.stringify(flaw.value) !== JSON.stringify(initialFlaw.value),
```

Comment:
Changes to affects cause this to return true. If the performance is acceptable, ramda is convenient for this case
```suggestion
  () => R.equals( 
    R.omit(['affects', 'acknowledgments', 'comments', 'references', 'cvss_scores'], flaw.value),
    R.omit(['affects', 'acknowledgments', 'comments', 'references', 'cvss_scores'], initialFlaw),
  )
```

Reply:
Another approach would be [using a callback with `JSON.stringify`](https://stackoverflow.com/a/46908358) to limit traversal to one level, if that suffices: it would be more performant than using ramda.

Reply:
> Changes to affects cause this to return true. If the performance is acceptable, ramda is convenient for this case
Oh you are right, I must have change something after because I remember trying this and it worked 🤔 

I think the callback approach will be enough